frontend/packages/console-dynamic-plugin-sdk/src/models/index.ts

```diff
@@ -0,0 +1,32 @@
+import { K8sKind } from '../api/common-types';
+
+export const ProjectModel: K8sKind = {
+  apiVersion: 'v1',
+  apiGroup: 'project.openshift.io',
+  label: 'Project',
+  // t('public~Project')
+  labelKey: 'public~Project',
+  plural: 'projects',
+  abbr: 'PR',
+  kind: 'Project',
+  id: 'project',
+  labelPlural: 'Projects',
+  // t('public~Projects')
+  labelPluralKey: 'public~Projects',
+};
+
+export const SelfSubjectAccessReviewModel: K8sKind = {
+  label: 'SelfSubjectAccessReview',
+  // t('public~SelfSubjectAccessReview')
+  labelKey: 'public~SelfSubjectAccessReview',
```

Comment:
I see this is needed as the model is used in `k8sCreate`,  can have a minimal copy if really needed i.e group/version/kind/plural, and avoid translation stuff as well.